resources/content/articles/gemeinschaft_johanniter.html

```diff
@@ -0,0 +1,36 @@
+---
+title: Johanniter
+link: gemeinschaft_johanniter
+category: Allgemein
+author: 'Max Mustermann'
+thumbnail: wirtschaft_neues_aus_wr_thumbnail
+thumbnail_slide: wirtschaft_neues_aus_wr_slide_image
+published_on: 02.11.2021 00:12
+---
+
+<h3>Schüler und Schülerinnen des WGG setzen ein Zeichen des Zusammenhalts</h3>
+<p>
+    Seit 1993 packen Tausende Kindergartenkinder, Schülerinnen und Schüler, Mitarbeitende von Unternehmen und viele
+    Privathaushalte alljährlich in der Vorweihnachtszeit Hilfspakete für notleidende Menschen in Südosteuropa. Heuer
+    machen auch die Schülerinnen und Schüler des Willibald-Gluck-Gymnasiums mit.
+</p>
+<figure class="float-left">
+    <img src="img/gemeinschaft_johanniter_zeichen.jpg">
```

Comment:
this is more like you should implement pictures, with the `figure` tag, but you should still use `v-image`